### PR TITLE
fixed misleading description of geoserver deployment

### PR DESCRIPTION
**Pull request by thomasstorm**

### CODE REVIEW

File: docker/geoserver/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,14 @@
-# An image usoing this docker file has been pushed to quay.io/bcdev/xcube-geodb-backend:GEODB_VERSION
-# This image is automatically built using q quay build trigger
+# An image on basis of this docker file should be pushed to quay.io/bcdev/xcube-geoserv:<version>
+# The version may diverge from geodb version as geoserver is expected to seldom change.
+# After pushing a new version of this docker image to quay.io, adapt tag in k8s-configs/xcube-geodb/helm/charts/values-*.yaml
+#
+# Example for version 1.0.3:
```
Comment (AliceBalfanz) [inline docker/geoserver/Dockerfile:5]:
```suggestion
# Example how to push a new docker image to quay.io for version 1.0.3:
```

**Review by AliceBalfanz — CHANGES_REQUESTED** [review]
Please see my comments in the review. Please also investigate why CI test for nbs :)

**Comment by AliceBalfanz** [comment]
Hi Thomas, you manually updated the geoserver docker image - however we have a github action which does this automatically, as it seams in the CI:
image
We should either delete the gha for this (not my preferred option, because we would move a step back) , or use it in the future.

**Comment by thomasstorm** [comment]
> 

No - this action builds the [xcube-geodb-backend](https://quay.io/repository/bcdev/xcube-geodb-backend) image. That's different from the GeoServer image.

**Review by AliceBalfanz — APPROVED** [review]
Thanks for the response to my question :)

**Pull request merged**